packages/derived-wallet-solana/src/SolanaDerivedPublicKey.ts

```diff
@@ -41,7 +40,7 @@ export class SolanaDerivedPublicKey extends AccountPublicKey {
     const serializer = new Serializer();
     serializer.serializeBytes(utf8EncodedDomain);
     serializer.serializeFixedBytes(solanaPublicKeyBytes); // fixed length 32 bytes
-    const accountIdentifier = hashValues([serializer.toUint8Array()]);
+    const accountIdentifier = serializer.toUint8Array();
```

Comment:
Not hashing this anymore, so it can be deserialized in the authentication function to obtain the domain and Solana public key